packages/plugin-core/src/utils/StartupUtils.ts

```diff
@@ -490,6 +490,25 @@ export class StartupUtils {
     }
   }
 
+  static showUninstallMarkdownLinksExtensionMessage() {
+    if (VSCodeUtils.isExtensionInstalled("dendron.dendron-markdown-links")) {
+      vscode.window
+        .showInformationMessage(
+          "Please uninstall the Dendron Markdown Links extension. This extension is no longer maintained and Dendron has the note graph feature built-in natively now.",
```

Comment:
can we strengthen the wording to say 

"Please uninstall the Dendron Markdown Links extension. Dendron has the note graph feature built-in now and having this legacy extension installed will interfere with its functionality."